lib/src/models/receipts.dart

```diff
@@ -44,7 +44,7 @@ class Receipt {
   const Receipt(this.user, this.time);
 
   @override
-  bool operator ==(dynamic other) => (other is Receipt &&
+  bool operator ==(other) => (other is Receipt &&
```

Comment:
What did the analyzer warning actually say? Are you sure it didn't tell you to take an Object instead of dynamic here?

Reply:
the default when no type is specified here is `Object`. so, didn't change it.

Reply:
done